libs/core/src/components/sage-avatar/stories/sage-avatar.docs.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './sage-avatar.stories.js';
+
+<Meta title="components/Avatar" component="sage-avatar" argTypes={extractArgTypes('sage-avatar')} />
+
+# Avatar
+
+Avatar is used for showing a thumbnail representation of an admin, customer, or business.
+
+## Properties
+
+<ArgsTable of="sage-avatar" />
+
+### Default
```

Comment:
The default and admin stories are showing up the same for me and the props are not populating in the admin example:

<img width="1052" alt="Screenshot 2023-06-05 at 11 14 26 AM" src="https://github.com/Kajabi/sage/assets/1175111/c01300b3-708c-4385-bb1e-644be4d0f5ef">

Reply:
Minor bug with the admin story, this is fixed in the coming commits.